### ISSUE TITLE
Catch with_items loop task without use of {{ item }}

### ISSUE THREAD

**Issue by leynos** [ansible/ansible-lint#256]
# Issue Type
- Feature request

# Ansible and Ansible Lint details

```
ansible 2.2.2.0
  config file =
  configured module search path = Default w/o overrides
ansible-lint 3.4.12
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour

Sometimes, it can be a bit of a headscratcher when Ansible reports a directory as not existing when the directory in question has been included in a `with_items` loop task to create a bunch of directories.  It's only after a fair amount of searching that one notices what should be obvious, that the `{{ item }}` variable has never actually been referenced in the task.

I'm sure this has happened to others at one point, and I think it would be a good help to have this caught before it causes anguish.

**Comment by willthames** [comment]
Someone would need to come up with a suitable mechanism for catching this (including loop vars that aren't called `item`)

**Comment by ssbarnea** [comment]
Closing as it seems obsolete but if you think the bug is real, just make PR with an xfail marked test that reproduces it.
